templates/adherent_profile/informations/elections.html.twig

```diff
@@ -0,0 +1,95 @@
+{% extends 'adherent_profile/_layout.html.twig' %}
+
+{% block adherent_profile_content %}
+    <section class="adherent-profile__section">
+        <h1 class="text--medium-large">Élections</h1>
+
+        <nav class="tabs-nav b__nudge--bottom-larger">
+            <ul>
+                <li class="active"><a href="#" class="tabs-nav__item">Mandats</a></li>
+                <li><a href="#" class="tabs-nav__item">Candidatures</a></li>
+                <li><a href="#" class="tabs-nav__item">Étiquettes</a></li>
+                <li><a href="#" class="tabs-nav__item">Fonctions</a></li>
+                <li><a href="#" class="tabs-nav__item">Parrainages</a></li>
+            </ul>
+        </nav>
+
+        <h2 class="section-subtitle">Vos mandats</h2>
+
+        <div class="text--body text--black text--bold b__nudge--none b__nudge--bottom-large">
+            Vos mandats en cours
+        </div>
+        <div class="data-display">
+            <div class="data-display__item">
+                <div class="label">Périmètre géographique</div>
+                <div class="value">1ère circonscription du Val-de-Marne</div>
+            </div>
+            <div class="data-display__item">
+                <div class="label">Label</div>
```

Comment:
Is a content of this page is temporary?

Reply:
this page is integrated by Enzo, but the feature is not ready